templates/cisco_ios_show_ip_interface.template

```diff
@@ -48,8 +48,10 @@ Start
   ^\s+Policy
   ^\s+Network\s+address\s+
   ^\s+BGP
+  ^\s+Sampled Netflow
```

Comment:
Do you mind removing the static spacing and use either **\s*** or **\s+**. Once those are implemented, we can get this merged.

Reply:
Should be fixed.